_posts/2022-07-24-opensearch-plugin-zips-now-in-maven-repo.markdown

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title:  "OpenSearch plugin zips now in Maven repo"
+authors: 
+  - prudhvigodithi
+date: 2022-07-24 01:01:01 -0700
+categories: 
+  - technical-post
+twittercard:
+  description: "Details on how to consume OpenSearch plugin zips from maven repo and the process involved in shipping them to the maven repo."
+---
+
```

Comment:
I deleted the Summary section and used that description as the Intro.

```suggestion
Starting with the release of OpenSearch `2.1.0`, OpenSearch plugin zips are now signed and published to a central Apache Maven [repo](https://repo1.maven.org/maven2/org/opensearch/plugin/).  Using the [Release zips Maven Repo](https://repo1.maven.org/maven2/org/opensearch/plugin/) and [Snapshot zips Maven Repo](https://aws.oss.sonatype.org/content/repositories/snapshots/org/opensearch/plugin/) URL’s, OpenSearch plugin zips can now be consumed  as a dependency to build other plugins or fetched as standalone components for your OpenSearch cluster.
```

Reply:
Sentence Reformation:

Starting with the release of OpenSearch `2.1.0`, OpenSearch plugin zips are now signed and published to a central Apache Maven [repo](https://repo1.maven.org/maven2/org/opensearch/plugin/). Using the [Release zips](https://repo1.maven.org/maven2/org/opensearch/plugin/) and [Snapshot zips](https://aws.oss.sonatype.org/content/repositories/snapshots/org/opensearch/plugin/) Maven Repo URL’s, OpenSearch plugin zips can now be consumed  as a dependency to build other plugins or fetched as standalone components for your OpenSearch cluster.